cypress.config.js

```diff
@@ -5,4 +5,4 @@ module.exports = defineConfig({
     setupNodeEvents(on, config) {
     },
   },
-})
+})
\ No newline at end of file
```

Comment:
Add a newline at the end of the file for better readability and to follow best practices